Sources/VaporToolbox/Build.swift

```diff
@@ -25,107 +30,100 @@ public final class Build: Command {
     }
 
     public func run(arguments: [String]) throws {
-        if arguments.options["clean"]?.bool == true {
-            let clean = Clean(console: console)
-            try clean.run(arguments: arguments)
-        }
+        try clean(arguments)
+        try fetch(arguments)
+        try build(arguments)
+        try run(arguments)
+    }
 
-        if arguments.options["fetch"]?.bool != false {
-            let fetch = Fetch(console: console)
-            try fetch.run(arguments: [])
-        }
+    private func clean(_ arguments: [String]) throws {
+        guard arguments.flag("clean") else { return }
+        let arguments = arguments.removeFlags(["clean"])
```

Comment:
should arguments be `inout`? since clean has taken care of this flag it shouldn't really be propagated